src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -1327,11 +1327,11 @@ static CHIP_ERROR OpenCommissioningWindow(Controller::DeviceController * control
 
 } // anonymous namespace
 
-- (void)openCommissioningWindowWithSetupPasscode:(NSNumber *)setupPasscode
-                                   discriminator:(NSNumber *)discriminator
-                                        duration:(NSNumber *)duration
-                                           queue:(dispatch_queue_t)queue
-                                      completion:(MTRDeviceOpenCommissioningWindowHandler)completion
+- (void)_openCommissioningWindowWithSetupPasscode:(nullable NSNumber *)setupPasscode
+                                    discriminator:(NSNumber *)discriminator
```

Comment:
are these whitespaces before the params?

Reply:
Running restyle fixed this.